packages/e2e-tests/specs/experiments/document-settings.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	trashAllPosts,
+	activateTheme,
+} from '@wordpress/e2e-test-utils';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { navigationPanel } from '../../experimental-features';
+
+async function getDocumentSettingsTitle() {
+	const titleElement = await page.waitForSelector(
+		'.edit-site-document-actions__title'
+	);
+
+	return titleElement.evaluate( ( el ) => el.innerText );
+}
+
+async function getDocumentSettingsSecondaryTitle() {
+	const secondaryTitleElement = await page.waitForSelector(
+		'.edit-site-document-actions__secondary-item'
+	);
+
+	return secondaryTitleElement.evaluate( ( el ) => el.innerText );
+}
+
+describe( 'Document Settings', () => {
+	beforeAll( async () => {
+		await activateTheme( 'tt1-blocks' );
+		await trashAllPosts( 'wp_template' );
+		await trashAllPosts( 'wp_template_part' );
+	} );
+	afterAll( async () => {
+		await trashAllPosts( 'wp_template' );
+		await trashAllPosts( 'wp_template_part' );
+		await activateTheme( 'twentytwentyone' );
+	} );
+
+	beforeEach( async () => {
+		await visitAdminPage(
+			'admin.php',
+			addQueryArgs( '', {
+				page: 'gutenberg-edit-site',
+			} ).slice( 1 )
+		);
+		await page.waitForSelector( '.edit-site-visual-editor' );
```

Comment:
We have a helper function for this as well.

```js
import { siteEditor } from '../../experimental-features';
```

```suggestion
		await siteEditor.visit();
```

Reply:
I'll change this as well